docs/developer-cookbook.rst

```diff
@@ -438,43 +438,26 @@ by the environment variable ``GIRDER_TEST_DATA_PREFIX`` as follows
     with open(test_file, 'r') as f:
         content = f.read() # The content of the downloaded test file
 
-Serving a custom app from the server root
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Mounting a custom application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Normally, the root node (``/``) of the server will serve up the Girder web client.
-Some plugins will wish to change this so that their own custom app gets served out of
-the server root instead, and they may also want to move the Girder web client to
-be served out of an alternative route so they can still use it in addition to
-their custom front-end application.
+A plugin may contain an entire application separate from the default Girder
+web client. This plugin may be written in a way which enables administrators
```

Comment:
What does it mean that the "plugin may be written in a way which enables administrators to mount the application" ?  Is the previous sentence sufficient?  This sentence I'm asking about makes me think my plugin would have to do something specific beyond providing an entire web application.